spec/ffi/rbx/attach_function_spec.rb

```diff
@@ -35,4 +35,4 @@ module LibC
       expect(t[:tv_usec]).to be_kind_of(Numeric)
     end
   end
-end
+end unless RUBY_ENGINE == 'jruby'
```

Comment:
on Windows, gettimeofday doesn't exist in the standard libc library, but cruby redefines it. That's why it cannot be supported on jruby.